etcdctl/ctlv3/command/get_command.go

```diff
@@ -112,44 +123,56 @@ func getGetOp(args []string) (string, []clientv3.OpOption) {
 		opts = append(opts, clientv3.WithRange(args[1]))
 	}
 
-	opts = append(opts, clientv3.WithLimit(getLimit))
+	if getLimit != 0 {
```

Comment:
Can `getLimit`  be negative?

Reply:
seems we did not care much about this...all options provided by clientv3 uses int64
```go
func WithLimit(n int64) OpOption ...
func WithRev(rev int64) OpOption ...
```

Reply:
Why not `if getLimit > 0` ?

Reply:
yeah, u are right..